packages/core/echo/echo-pipeline/src/metadata/metadata-store.ts

```diff
@@ -247,6 +248,29 @@ export class MetadataStore {
     await this.flush();
   }
 
+  getPersistentInvitations(): Invitation[] {
```

Comment:
since the context does the "persistent" prefix add anything? vs. getInvitations.
i'm not sure. i.e., i would not expect to get non-persistent invitations from a store. same comment throughout about decoupling the name.